### PR TITLE
connected React routers to components

### PR DESCRIPTION
**Pull request by christopher-r-morehead**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -58,9 +62,22 @@ const App = () => {
...
+            <Route path="/SignUp">
+              <SignUp />
+            </Route>
+            <Route path="/Cart">
+              {/* <Cart /> */}
+            </Route>
+            <Route exact path="/">
+              <Products />
+            </Route> 
+          </Switch>
```
Comment (jendiorio) [inline src/components/App.js:80]: I think we should include `<Redirect to="/" />` before the final `</Switch>` tag, then any paths that don't exist will redirect to the products page.

File: src/components/App.js [hunk 2]
```diff
@@ -1,4 +1,8 @@
 import React, { useState, useEffect } from 'react';
+import {
+  Switch,
+  Route
```
Comment (jendiorio) [inline src/components/App.js:4]: Add `Redirect` to import per suggestion below.

File: src/components/App.js (hunk 1 above)
Comment (christopher-r-morehead) [reply src/components/App.js:82]: Didn't even think about that. Now added.

File: src/components/Header.js [hunk 3]
```diff
@@ -56,14 +65,17 @@ export default function ButtonAppBar() {
...
               color="inherit"
               aria-label="cart"
+              // content={Link} to="/Cart"
+              href="/Cart"
             >
               <Badge badgeContent={0} showZero color="secondary">
-                <ShoppingCartIcon />
+                <ShoppingCartIcon  />
               </Badge>
             </IconButton>
```
Comment (jendiorio) [inline src/components/Header.js:74]: I think you can wrap the entire component with `<Link>` to get that working? Ex: `<Link to="/Cart"><IconButton>...</IconButton></Link>`

**Pull request merged**